.circleci/config.yml

```diff
@@ -24,8 +24,10 @@ jobs:
 
       # We should also be able to test targets in a different workspace
       - run: bazel test --config=ci @program_example//...
-      # TODO(mattmoor): figure out why other tests in this example fail
       - run: cd examples/packages && bazel test //:test_escape_npm_json
+      - run: cd examples/packages && bazel test //:test_escape_npm_lock
```

Comment:
this isn't a good model - I wasn't able to get the Bazel CI to do this so these are missing windows coverage.
I'm starting to think that nested workspaces are more trouble than they are worth, given that we need some multi-repo CI setup anyway.